src/templates/kit/ClaimDomain.js

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useState } from 'react'
+import { Info, GU } from '@aragon/ui'
+import {
+  DOMAIN_CHECK,
+  DOMAIN_NONE,
+  DomainField,
+  Header,
+  PrevNextFooter,
+} from '.'
+
+function ClaimDomain({
+  back,
+  data,
+  next,
+  screenIndex,
+  screens,
+  screenTitle = 'Claim a domain',
+  screenSubtitle = 'Create your own organization and token in a few minutes!',
+}) {
+  const [domain, setDomain] = useState(data.domain || '')
+
+  const handleNext = useCallback(() => {
+    next({ domain })
+  }, [domain, next])
+
+  return (
+    <React.Fragment>
+      <Header title={screenTitle} subtitle={screenSubtitle} />
+
+      <DomainField
```

Comment:
Should check when updating too:

![image](https://user-images.githubusercontent.com/3072458/64346066-c32bb400-cff1-11e9-9e67-0498c937ccc9.png)

Reply:
Yes, it’s not connected to the domain checker yet.